app/views/application/_error_messages.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if object && object.errors.size>0 %>
```

Comment:
we can may be try to use safe-navigations here
```suggestion
<% if object&.errors %>
```

Reply:
Okay bhai. Modifying it. Thank you.